benchmarks/apache-spark/src/main/scala/org/renaissance/apache/spark/Als.scala

```diff
@@ -22,14 +23,22 @@ import scala.util.Random
 @Summary("Runs the ALS algorithm from the Spark MLlib.")
 @Licenses(Array(License.APACHE2))
 @Repetitions(30)
-class Als extends RenaissanceBenchmark with SparkUtil {
+@Parameter(name = "rating_count", defaultValue = "20000")
+// Work around @Repeatable annotations not working in this Scala version.
+@Configurations(
+  Array(
+    new Configuration(name = "test", settings = Array("rating_count = 500")),
+    new Configuration(name = "jmh")
+  )
+)
+final class Als extends Benchmark with SparkUtil {
 
   // TODO: Consolidate benchmark parameters across the suite.
   //  See: https://github.com/renaissance-benchmarks/renaissance/issues/27
 
-  var numRatings = 20000
+  private var ratingCountParam: Int = _
 
-  val THREAD_COUNT = 4
+  private val THREAD_COUNT = 4
```

Comment:
Should we add a parameter `thread_count` here?

Reply:
I don't have a problem with that. As I mentioned elsewhere, I only added configurable parameters for things that were being changed in response to "functionalTest" check.